.github/workflows/tests.yaml

```diff
@@ -19,4 +19,8 @@ jobs:
     - name: Get dependencies
       run: go get -v -t -d ./...
     - name: Run tests
-      run: go test main_test.go main.go
+      run: go test -coverprofile=cover.out main_test.go main.go
+    - name: Install Vim
+      run: apt-get update; apt-get install -y vim;
```

Comment:
Why vim?

Reply:
it's required by the action, it usually comes with the os but the ci worker is a container with almost nothing in it :)

the action uses `ex` to replace some text from go's default html coverage

Reply:
Ok

Reply:
> the action uses ex to replace some text from go's default html coverage

Why `ex` and not  `sed`?

Also, if you really need `vim`, I would suggest using `vim-nox` package